release_notes_generator/action_inputs.py

```
@@ -178,6 +182,40 @@ def get_release_notes_title() -> str:
+    def get_coderabbit_release_notes_title() -> str:
+        """
+        Get the CodeRabbit release notes title from the action inputs.
+        """
+        return get_action_input(CODERABBIT_RELEASE_NOTES_TITLE, CODERABBIT_RELEASE_NOTE_TITLE_DEFAULT)  # type: ignore[return-value]
+
+    @staticmethod
+    def get_coderabbit_summary_ignore_groups() -> list[str]:
+        """
+        Get the CodeRabbit summary title types to ignore.
+        """
+        l: list[str] = []
```

Ruff (0.11.9)

Ambiguous variable name: `l`

(E741)